source/future.tex

```diff
@@ -762,6 +777,21 @@
 Equivalent to: \tcode{return visit(std::forward<Visitor>(vis), arg.value);}
 \end{itemdescr}
 
+\rSec1[depr.ctime]{Deprecated time formatting}
```

Comment:
The paper titles this subclause

> Header `<ctime>` synopsis

I assume the change is intentional?

Reply:
No the paper has "D.? Deprecated [depr.ctime]" and then "D.?.? Header <ctime> synopsis [depr.ctime.syn]". As noted in the commit msg, the D.?.? subclause has been dissolved, so there's only D.? here, so strictly speaking the paper titled it "Deprecated".

"Deprecated" is a terrible name (I think I meant to write something there and apparently forgot).  And "Header <ctime> synopsis" only made sense for an rSec2 title, not the rSec1 title. So this change seems necessary.